src/test/java/hw8/SpellerTest.java

```diff
@@ -0,0 +1,68 @@
+package hw8;
+
+import hw8.dto.ResponseDTO;
+import hw8.dto.TextDTO;
+import hw8.dto.TextsDTO;
+import hw8.services.ActionService;
+import hw8.services.AssertionService;
+import io.restassured.response.Response;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class SpellerTest {
+
+    private AssertionService assertionService;
+    private ActionService actionService;
+
+    @BeforeClass
+    public void SetUp() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.